scripts/gulp/lint.js

```diff
@@ -63,12 +60,22 @@ gulp.task( 'lint:styles', () => {
     } ) );
 } );
 
+/**
+ * Lints the release js for errors.
+ */
+gulp.task( 'lint:release', () => {
```

Comment:
Could this be included in the `lint:build` task? Do we need a separate task for this one?

Reply:
@anselmbradford I could see that. For speed and clarity right now I'd like to keep them separated while I get the release process tested, but I can make a todo to combine them once that's complete.

Reply:
👍